internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -224,3 +236,27 @@ func envVarOutputNames(outputs []addon.Output) []string {
 	}
 	return envVars
 }
+
+// Validate that paths contain only an approved set of characters to guard against command injection.
+// We can accept 0-9A-Za-z-_.
+func validatePath(input string, maxLength int) error {
+	if len(input) > maxLength {
+		return fmt.Errorf("path must be less than %d bytes in length", maxLength)
+	}
+	if len(input) == 0 {
+		return nil
+	}
+	m := pathRegexp.FindStringSubmatch(input)
+	if len(m) == 0 {
+		return fmt.Errorf("paths can only contain the characters a-zA-Z0-9.-_/")
```

Comment:
Same here; maybe something like this?
```suggestion
		return fmt.Errorf("path %s can only contain the characters a-zA-Z0-9.-_/", input)
```